examples/vue/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "dev": "nodemon --watch ../dist/index.js -x \"cross-env DEBUG=vite-plugin-windicss:* vite --open\"",
+    "dev": "nodemon --watch ../../packages/vite-plugin-windicss/dist/index.js -x \"cross-env DEBUG=vite-plugin-windicss:devtools vite\"",
```

Comment:
Don't need nodemon anymore.

Reply:
no the sort restart still had some caching issues  😞